TheaterDatabase/dbo/Tables/Settings.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Settings] (
-    [SettingsId] INT identity(1,1) NOT NULL,
+
+    [SettingsId]  NVARCHAR(50) NOT NULL,
```

Comment:
why autoincrement removed?

Reply:
because now Settingid = its a device id

Reply:
it is  unique string type

Reply:
yes, it's a string type https://www.npmjs.com/package/react-native-device-info#getuniqueid